show/kdump.py

```diff
@@ -1,85 +1,200 @@
+import itertools
+import os
+import sys
+
 import click
+from tabulate import tabulate
+
 import utilities_common.cli as clicommon
 from swsscommon.swsscommon import ConfigDBConnector
 
 #
-# 'kdump command ("show kdump ...")
+# 'kdump' group (show kdump ...)
 #
 @click.group(cls=clicommon.AliasedGroup, name="kdump")
 def kdump():
-    """Show kdump configuration, status and information """
+    """Show kdump configuration, dump files and dmesg logs"""
     pass
 
-@kdump.command('enabled')
-def enabled():
-    """Show if kdump is enabled or disabled"""
-    kdump_is_enabled = False
+
+def get_kdump_config(field_name):
+    """Fetches the configuration of Kdump from `CONFIG_DB`.
+
+    Args:
+      field_name: A string contains the field name in the sub-table of 'config'.
+
+    Returns:
+      field_value: If field name was found, then returns the corresponding value.
+                   Otherwise, returns "Unknown".
+    """
+    field_value = "Unknown"
     config_db = ConfigDBConnector()
-    if config_db is not None:
+    if config_db:
```

Comment:
>if config_db [](http://example.com/codeflow?start=4&length=12)

If condition is false, is it an error?

Reply:
Yes, I think this will be an error.

Reply:
Then please handle the error instead of keeping silent. Or remove the check and let `config.connect()` throw exception.

Reply:
Updated and rely on the `config.connect()` to throw exception if any error occurs.